docs/source/deduplication.rst

```diff
@@ -0,0 +1,40 @@
+DeDuplication of EBU-TT-Live documents
+======================================
+
+When documents are ReSequenced, duplication of ``style`` and ``region`` elements
+and attributes can occur. To address this, the DeDuplicator node takes in a
+``manifest_file``, and processes the document(s) with the
+:py:func:`ebu_tt_live.node.deduplicator.DeDuplicatorNode.remove_duplication` function.
+
+After copying ``styling`` and ``layout`` into a ``list()`` and setting them up for new
+``style`` and ``region`` elements, respectively, the ``list()`` is passed through the
+:py:func:`ebu_tt_live.node.deduplicator.DeDuplicatorNode.CollateUniqueVals` function.
+
+Because ``style`` and ``region`` elements can have ``style`` attributes, these
+are deduplicated first. Each element is then passed through the
```

Comment:
Worth noting that a result of the deduplication might be two identical styles which differed only in the style attribute references before, but those references pointed at identical styles, so after, they end up looking the same. We don't do a repeat pass to deal with this, but that's a possible modification for later.